onnx-ops/README.md

```diff
@@ -0,0 +1,172 @@
+# ONNX Operator Tests
+
+This test suite exercises ONNX (Open Neural Network Exchange: https://onnx.ai/)
+operators (https://onnx.ai/onnx/operators/).
+
+Testing follows several stages:
+
+```mermaid
+graph LR
+  Import -. "\n(offline)" .-> Compile
+  Compile --> Run
+```
+
+Importing is run "offline" and the outputs are checked in to the repository for
+ease of use in downstream projects and by developers who prefer to work directly
+with `.mlir` files and native (C/C++) tools.
+
+## Quickstart
```

Comment:
@renxida I took some ideas from https://github.com/nod-ai/SHARK-TestSuite/pull/306 here when updating this README